website-profile.html

```diff
@@ -1 +1,54 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Lewis's Profile</title>
+<link rel="stylesheet" href="style.css">
+<body>
+
+<div class="top-banner">
+    <img class="lewis-ski-img" src="lewis-ski-img.jpeg" alt="Lewis on the ski slopes." style="float:right;width:300px;height:400px;">
+    <h1 class="header"> LEWIS MURRAY</h1>
+    <h2 class="subtitle"><i>"That's bless."</i></h2> 
+</div>
+
+<div class=“main”>
+    <div class="side">
+      <h2 class= "bio-header"> Bio</h2>
+      <p class="bio-text"> Hi, my name is Lewis Murray,<br>I am 20 years old and I am from Birmingham.<br>I currently work in construction and have been for 2 years and prior to that I worked in a pub.</p>
+      
+      <h2 class="personality-header">The good, the bad, the Lewis... personality traits</h2>
+      <p class= "personality-text">
+            I am sociable and friendly and I like to think that I can be quite funny sometimes.<br> 
+            I can be quite annoying sometimes but that can be seen as a bad thing.
+      </p>
+      <h2 class="perfect-day-header">Lewis's Perfect Day</h2>
+      <p class="perfect-day-text"><i>"I think an ideal day would be: in a foreign country, either somewhere really hot or at a skiing resort. Hitting the slopes or grabbing a drink and relaxing in the sun in the morning, then onto a party at night."</i>
+      </p>
+        <h2 class="hobbies-header"> Hobbies & Interests</h2>
+        <ul class="hobbies-text">
+            <li><strong>Favourite food:</strong> steak or enchiladas</li>
```

Comment:
Style in CSS to reduce HTML tags. You can remove the <strong></strong> from each li element by applying this in the CSS.
```suggestion
            .hobbies-text li {
                font-weight: bold;
           }
```